cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -106,22 +106,22 @@ const (
 secret_configs:
 - from:
     key-name-1:
-      bw_item: item-name-1
+      item: item-name-1
```

Comment:
Can we not change anything here to show that the existing config works as before?

Reply:
Let me add both cases then.